### ISSUE TITLE
Check ligatures are consistent

### ISSUE THREAD

**Issue by davelab6** [googlefonts/fontbakery#2747]
## Observed behaviour

https://twitter.com/mjdye/status/1223762539768467456?s=19

> So, I’m playing around with @ebensorkin 's Gelasio on google fonts and have noticed ligatures are weirdly inconsistent. See screenshot. Is that a mistake or is that intentional? https://t.co/UaXivIpZ1z

![IMG_20200201_200959](https://user-images.githubusercontent.com/261579/73601408-05b8d480-452f-11ea-8923-0015d97cb2e4.jpg)

## Expected behaviour

Ligatures should be technically checked for existing, and visually checked for similarities. The latter might be more of an exq thing

**Comment by felipesanches** [comment]
could you please clarify "exq thing"?

**Comment by felipesanches** [comment]
Are we discussing here a proposal of a check specific to `fi-fl-ffi-ffl` ligatures?

Or are we talking bout a broader issue that involves a larger set of potential ligatures?

**Comment by felipesanches** [comment]
also...

* **"checked for existing"** sounds like something easily scriptable as long as we have a list of pairs to check
* **"visually checked for similarities"** sounds like a task for actual human type-designers to do so it might be dashboard-level.

**Comment by davelab6** [comment]
Eqx, by sorkin type

**Comment by EbenSorkin** [comment]
Yeah this could be an EQX pass fail question or possibly 2+ basic questions which build to a family consistency question. I'll have a look at this post Merriweather and revise the source. For some reason I don't recall if I made Gelasio into a VF. If I somehow didn't I'll do that as well.
